vyper/exceptions.py

```diff
@@ -108,7 +108,7 @@ def __str__(self):
             if isinstance(node, vy_ast.VyperNode):
                 module_node = node.get_ancestor(vy_ast.Module)
 
-                if module_node.get("path") not in (None, "<unknown>"):
+                if module_node and module_node.get("path") not in (None, "<unknown>"):
```

Comment:
This prevents the compiler from throwing when trying to generate the error message for `folded_value` nodes since they do not have a module node.

Reply:
hmm maybe we should touch up exceptions for folded nodes? can we see a couple examples of what folded nodes look like